pkg/controller/controller_test.go

```diff
@@ -2138,42 +2181,38 @@ func TestUpdateBindingCondition(t *testing.T) {
 			name:                  "initially unset",
 			input:                 getTestBinding(),
 			status:                v1alpha1.ConditionFalse,
-			message:               "message",
 			transitionTimeChanged: true,
 		},
 		{
 			name:                  "not ready -> not ready",
 			input:                 getTestBindingWithStatus(v1alpha1.ConditionFalse),
 			status:                v1alpha1.ConditionFalse,
-			message:               "message",
+			transitionTimeChanged: false,
+		},
+		{
+			name:                  "not ready -> not ready, message and reason change",
```

Comment:
I might have missed it but is there a test that only changes one at a time message -> messageX and reason -> reasonY.